package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "llama_cloud_services",
+  "version": "1.0.0",
+  "private": "true",
+  "license": "MIT",
+  "scripts": {
```

Comment:
this main script should trigger `changeset version` (changeset works with monorepos and will update all versions for all package.jsons in the monorepo)